projects/discussion-ui/src/lib/components/discussion-details/discussion-details.component.html

```diff
@@ -53,12 +53,12 @@
                 <a href="javascript:void(0)" i18n-aria-label role="link" *ngIf="!post.upvoted" (click)="upvote(post);logTelemetry($event, post)"
                   aria-label="upvote the post" id="up-vote">
                   <!-- upward arrow -->
-                  <img src="./assets/discussion-ui/images/up-arrow.png" alt="up-arrow" width="14px" height="14px" class="icons">
+                  <img src="./assets/discussion-ui/images/up-arrow.png" width="14px" height="14px" class="icons">
```

Comment:
Why did you removed this "alt" property? Which is required on mouse-over to show the text. 
Get this property back for all the images

Reply:
This was removed by mistake while resolving conflicts